mandatory/3-function-output.js

```diff
@@ -1,17 +1,24 @@
 // Add comments to explain what this function does. You're meant to use Google!
+
+/* This function creat an integer random number(by multiplying decimal random number by ten)*/
 function getRandomNumber() {
   return Math.random() * 10;
 }
 
 // Add comments to explain what this function does. You're meant to use Google!
+
+/*This function concat two words which will be pass to the function as an argument
+for example if word1="ali" and word="reza" the result of this function will be "alireza"*/
 function combine2Words(word1, word2) {
   return word1.concat(word2);
 }
 
 function concatenate(firstWord, secondWord, thirdWord) {
-  // Write the body of this function to concatenate three words together.
-  // Look at the test case below to understand what this function is expected to return.
+  return firstWord + " " + secondWord + " " + thirdWord;
```

Comment:
Hello Leila,  you have done an amazing job and your codes are simple and easy to understand and review. However, I think for the return you could have used .concat to concatenate all the words together like on the above combine2Words function. If there is a reason for not using that do you mind explaining? Thank you

Reply:
Thank you for your feedback. Yes you are completely right. I just wanted to try every piece of code that we have learnt during course :)